Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -917,23 +917,21 @@ def fetch_last_emails(account, folder_name='Inbox', since_datetime=None, exclude
     qs = qs.filter().order_by('datetime_received')
 
     result = []
+    counter = 0
     for item in qs:
+        counter += 1
         try:
-            if isinstance(item, Message):
+            if isinstance(item, Message) and item.message_id not in exclude_ids:
```

Comment:
```suggestion
            if isinstance(item, Message) and not item.message_id in exclude_ids:
```
Better performance.

Reply:
https://www.flake8rules.com/rules/E713.html says not to.

Reply:
correct